lib/server.js

```diff
@@ -186,6 +187,40 @@ const createServer = ({
     })
   }
 
+  if (proxy) {
+    const expressProxy = require('express-http-proxy')
+    for (const [base, target] of Object.entries(proxy)) {
+      let handler
+
+      if (Array.isArray(target)) {
```

Comment:
I'd like to move this block to a utility function outside `createServer`, since it has no other deps on local variables. Something like `createProxyHandler`, the like of `formatHost` above. (TBH, also motivated by sparing the `let` :sweat_smile: )

Reply:
Done